applications/jupyter/workloads-auto-create-brand.example.tfvars

```diff
@@ -26,21 +26,23 @@ cluster_membership_id = "" # required only for private cluster, default: cluster
 #######################################################
 
 ## JupyterHub variables
-namespace              = "jupyter"
-create_gcs_bucket      = true
-gcs_bucket             = "<gcs-bucket>"
-create_service_account = true
-gcp_service_account    = "jupyter-service-account"
+namespace                                   = "jupyter"
+create_gcs_bucket                           = true
+gcs_bucket                                  = "<gcs-bucket>"
+workload_identity_service_account           = "jupyter-service-account"
 
 # Jupyterhub with IAP
-add_auth                = true
-brand                   = ""
-support_email           = "<email>"
-default_backend_service = "proxy-public"
-service_name            = "iap-config-default"
+add_auth                  = true
+brand                     = "projects/<prj-number>/brands/<prj-number>" # ensure brand is Internal
```

Comment:
If this is the "auto-create-brand" file example, do we still need this template? If we're just going to have the same values/vars between the tfvar files I don't see the value added.

Reply:
Oops, good catch.